documentation/starter-kits/CONTRIBUTING/CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+
+Resolving bugs is a priority for our project. We welcome bug reports. However, please make sure to do the following prior to submitting a bug report:
+- **Check for duplicates** - there may be a bug report already describing your issue, so check the issue tracking system first
+
+Here's some guidance on submitting a bug issue:
+1. Navigate to our issue tracking system and file a new issue
+2. Select a bug template (if available) for your issue
+   1. Fill out the template fields to the best of your ability, including output snippets or screenshots where applicable
+3. Follow the general guidelines below for extra information about your bug
+   1. Include a code snippet if you have it showcasing the bug
+   2. Provide reproducible steps of how to recreate the bug
+   3. If the bug triggers an exception or error message, include the *full message* or *stacktrace*
+   4. Provide information about your operating system and the version of our project you're using
+
+#### Submitting New Feature Issues
+
+We welcome new feature requests to help grow our project. However, please make sure to do the following prior to submitting a bug report:
+- **Check for duplicates** - there may be a new feature issue already describing your issue, so check the issue tracking system first
+- **Consider alternatives** - is your feature really needed? Or is there a feature within our project or with a third-party that may help you achieve what you want?
+
+Here's some guidance on submitting a new feature issue:
+1. Navigate to our issue tracking system and file a new issue
+2. Select a new feature template (if available) for your issue
+   1. Fill out the template fields to the best of your ability
+3. Follow the general guidelines below for extra information about your bug
+   1. Include a code snippet if you have it showcasing the bug
+   2. Provide reproducible steps of how to recreate the bug
+   3. If the bug triggers an exception or error message, include the *full message* or *stacktrace*
+   4. Provide information about your operating system and the version of our project you're using
+
+
+#### Submitting Security Vulnerability Issues
+
+Security vulnerabilities should **not** be filed to the regular issue tracking system.
+
+Report your security vulnerabilities by: [INSERT SECURITY VULNERABILITY E-MAIL / SUBMISSION FORM HERE]
+
+Please be sure to:
+* Indicate the severity of the vulnerability
+* Provide any workarounds, if you know them
+* Provide return-contact information to follow-up with you if needed
+
+#### Reviewing Pull Requests
+
+Reviewing others' contributions is a great way to learn about best practices in both contributions as well as software. 
+
+Take a look at our pull requests tracking system [INSERT LINK FOR PULL REQUESTS TRACKING SYSTEM], and try the following options for providing a review:
+1. Read the code / patch associated with the pull-request, and take not of any coding, bug, or documentation abnormalities if found
```

Comment:
Bullet 1 is oddly worded. I think the intent was "do not take any code, bug, or documentation with abnormalities"

Reply:
@jeffreypon - 😆 I wasn't trying to be Shakespearean, though I'll make a "note" of that for the future 😉 .

I was missing an `e` in the word `not`. I'll correct.